MEETINGS.md

```diff
@@ -339,6 +340,8 @@
 [Four Seasons Summer]: http://www.basilurtea.com.au/tea_collection/four_seasons/four-seasons-packet-lt-summer-tea-100g.html
 [Momo Oolong]: https://usa.lupicia.com/category/select/cid/308/pid/9383/language/en
 [Fruit Shop]: http://www.basilurshop.com.au/basilur/festive-collection-100g-lt-fruit-shop
+[Dragon well]: https://en.wikipedia.org/wiki/Longjing_tea
+[longjing]: https://en.wikipedia.org/wiki/Longjing_tea
```

Comment:
+[Dragon well]: https://en.wikipedia.org/wiki/Longjing_tea
+[longjing]: https://en.wikipedia.org/wiki/Longjing_tea
Why are you adding these links and not using them anywhere?